apps/pushy/src/pushy_node_stats_scanner.erl

```diff
@@ -48,7 +48,7 @@ start_link() ->
     gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
 
 init([]) ->
-    _Seed = random:seed(erlang:now()),
+    _Seed = random:seed(erlang:monotonic_time(milli_seconds)),
```

Comment:
I do think we want the old format here (rather than the milli_seconds arg) but the section `Seed Random Number Generation with a Unique Value` recommends using monotonic_time, I believe because os:timestamp() doesn't have the same guarantees as erlang:now() did.  I'm not sure whether we care about those here.